protractor.conf.js

```diff
@@ -17,9 +17,19 @@ exports.config = {
   jasmineNodeOpts: {
     showColors: true,
     defaultTimeoutInterval: 30000,
-    print: function() {}
+    print: function () { }
   },
   onPrepare() {
+    var HtmlReporter = require('protractor-beautiful-reporter');
+    // Add a screenshot reporter and store screenshots to `/tmp`:
+      jasmine.getEnv().addReporter(new HtmlReporter({
+         baseDirectory: 'tmp',
```

Comment:
baseDirectory name should be more appropriate like test-reports

Reply:
if its inside e2e then we can  simply  name it `report` else `e2e-test-report`